packaging_automation/update_package_properties.py

```diff
@@ -348,4 +349,5 @@ def update_all_changes(github_token: non_empty(non_blank(str)), package_properti
         run(f'git push --set-upstream origin {pr_branch}')
         create_pr(arguments.gh_token, pr_branch, commit_message, REPO_OWNER, PROJECT_NAME,
                   project.value.packaging_branch)
-        remove_cloned_code(execution_path)
+        if not arguments.pipeline:
```

Comment:
Why do we not remove the cloned code on pipelines? Any particular reason?

Reply:
because pipeline machines are disposable. So there is no need to delete the code there since machine is disposed at the end of the pipeline execution

Reply:
If an operation does not take too much time to complete, we can skip the conditionals and perform it everywhere. I think this hurts readability and we should revert this change here.

Reply:
Fixed